src/config.ts

```diff
@@ -77,3 +82,63 @@ export const setConfig = (newConfig: ConfigOpts): Config => {
     ...newConfig,
   });
 };
+
+// - Update all `getConfig()` calls to use these getters where appropriate
+// - Move associated "Missing" errors to the config module
+export const requireGetProvider = (opts?: ConfigOpts): ethers.providers.Provider => {
```

Comment:
Can you throw some typedoc comments on these methods?

Reply:
Yes, I meant actually to make this a draft PR and forgot.

Reply:
ah, oops